KIPs/kip-7-fungible_token.md

```diff
@@ -119,6 +120,79 @@ interface IKIP7 {
     /// @param recipient The owner will receive the tokens.
     /// @param amount The token amount will be transferred.
     function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+    
+    /// @notice Moves `amount` tokens from the caller's account to `recipient`.
+    /// @dev Throws if the message caller's balance does not have enough tokens to spend.
+    /// Throws if the contract is pausable and paused.	
+    /// Throws if `_to` is the zero address. 
+    /// Returns a boolean value indicating whether the operation succeeded.
+    /// When transfer is complete, this function checks if `_to` is a smart 
+    /// contract (code size > 0). If so, it calls
+    ///  `onKIP7Received` on `_to` and throws if the return value is not
+    ///  `bytes4(keccak256("onKIP7Received(address,address,uint256,bytes)"))`.
+    /// @param recipient The owner will receive the tokens.
+    /// @param amount The token amount will be transferred.
+    /// @param data Additional data with no specified format, sent in call to `_to`
+    function safeTransfer(address recipient, uint256 amount, bytes data) external;
+    
+    
+    /// @notice  Moves `amount` tokens from the caller's account to `recipient`.
+    /// @dev This works identically to the other function with an extra data parameter,
+    ///  except this function just sets data to "".
+    /// @param recipient The owner will receive the tokens.
+    /// @param amount The token amount will be transferred.
+    function safeTransfer(address recipient, uint256 amount) external;
```

Comment:
Does this function return something? If it does, we need to specify the return format like the tag in other function, `/// Returns a boolean value indicating whether the operation succeeded.`